core/modules/user/user.module

```diff
@@ -319,6 +319,23 @@ function user_save($account) {
   return $account->save();
 }
 
+/**
+ * Set a default value if one-time login link goes out,
```

Comment:
Can we add a 'Validation Handler' heading at the top of this docblock? (with a line break)  like we do on [node_form_validate](https://api.backdropcms.org/api/backdrop/core%21modules%21node%21node.pages.inc/function/node_form_validate/1)